ansible/templates/import_ips_for_large_providers.sh.j2

```diff
@@ -14,3 +14,6 @@ source .venv/bin/activate
 dotenv run -- ./manage.py update_networks_in_db_amazon
 dotenv run -- ./manage.py update_networks_in_db_google
 dotenv run -- ./manage.py update_networks_in_db_microsoft
+
+# TODO: activate once we have found our technical contact to talk to
+# dotenv run -- ./manage.py update_networks_in_db_cloudflare
```

Comment:
this is the command we would run on weekly import lime we do with AWS, GCP, and Microsoft